src/libraries/System.Net.Quic/src/System/Net/Quic/Implementations/MsQuic/MsQuicConnection.cs

```diff
@@ -707,6 +707,12 @@ private void Dispose(bool disposing)
 
             if (NetEventSource.Log.IsEnabled()) NetEventSource.Info(_state, $"{TraceId()} disposing {disposing}");
```

Comment:
Just glanced at this and noticed that this is against rule 1 of the [style guide](https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/coding-style.md). What's inside the block should be indented and put on its own line.

Reply:
we make exception for tracing to make it more compact. Unlike general code it is unlikely somebody would try to add more to the statement and miss it.

Reply:
Oh ok. Thanks. That's good to know!